hero-z-7045-builder

```diff
@@ -25,7 +25,14 @@ if [[ ! -f "${0##*/}" ]]; then
 fi
 
 update_env() {
+    git submodule update --init
```

Comment:
Please change this, inside `update_env()` we should really only do the update of the environment variables. But for sure we should not update submodules and the like. This function is executed whenever the builder is started.

Reply:
I agree about that, I will fix it.